client/pages/verify.js

```diff
@@ -0,0 +1,218 @@
+/* eslint-disable no-console */
+// Module imports
+import React from 'react'
+// import PropTypes from 'prop-types'
```

Comment:
remove unused import please.


also please space the top of this file out. 5 spaces between external and internal imports, and then another 5 between imports and the class.